website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -425,6 +425,7 @@ export default {
     async questAccept (partyId) {
       const quest = await this.$store.dispatch('quests:sendAction', { groupId: partyId, action: 'quests/accept' });
       this.user.party.quest = quest;
+      this.group.quest.members = quest.members;
```

Comment:
Hi @rogesson this code is working well but I noticed there's another small bug, if you're the last member to accepts the quest (so that after you accept it it should start) it doesn't show up as started until you refresh the page. I think it's happening because `this.group.quest` is not updated , I've not tested it but I feel like adding  `this.group.quest = quest;` should fix it

Reply:
Yeaah! I will try it, thanks!